src/pages/resources/custom-containers.mdx

```diff
@@ -0,0 +1,182 @@
+export const description = 'Customize how functions are built into containers'
+
+# Custom containers
+
+Nitric builds applications by identifying its entrypoints, which are typically defined in the `nitric.yaml` file as `handlers`. Each entrypoint in a Nitric app is built into its own container using Docker, then deployed to a cloud container runtime such as AWS Lambda, Google CloudRun or Azure Container Apps.
+
+The Nitric CLI decides how to build those containers based on the programming language used by the entrypoint, for example if the entrypoint is a python file it will be build using Nitric's python dockerfile template. These dockerfile templates are designed with compatibility and ease of use in mind, this makes building applications convenient but may not provide additional dependencies your code relies or the ideal optimization for your application.
```

Comment:
```suggestion
The Nitric CLI decides how to build those containers based on the programming language used by the entrypoint, for example if the entrypoint is a python file it will be built using Nitric's python dockerfile template. These dockerfile templates are designed with compatibility and ease of use in mind, this makes building applications convenient but may not provide additional dependencies your code relies on or the ideal optimization for your application.
```